src/quest_manager/views.py

```
@@ -1156,6 +1158,8 @@
                 draft_comment.target_object = submission
                 # reset timestamp needed otherwise it will use the draft comment's timestamp from when the submission was started
                 draft_comment.timestamp = timezone.now()
+                if paste_detected == True:
```

Ruff (0.8.2)

Avoid equality comparisons to `True`; use `if paste_detected:` for truth checks

Replace with `paste_detected`

(E712)